ngraph/frontend/frontend_manager/include/frontend_manager/place.hpp

```diff
@@ -94,12 +94,62 @@ namespace ngraph
             /// place
             virtual std::vector<Ptr> get_consuming_operations(int outputPortIndex) const;
 
+            /// \brief Returns references to all operation nodes that consume data from this place
+            /// for specified output port
+            ///
+            /// \note It can be called for any kind of graph place searching for the first consuming
+            /// operations.
+            ///
+            /// \param outputName If a given place is itself an operation node, this specifies name
+            /// of output port group
+            ///
+            /// \return A vector with all operation node references that consumes data from this
+            /// place
+            virtual std::vector<Ptr>
+                get_consuming_operations(const std::string& outputPortName) const;
+
+            /// \brief Returns references to all operation nodes that consume data from this place
+            /// for specified output port
+            ///
+            /// \note It can be called for any kind of graph place searching for the first consuming
+            /// operations.
+            ///
+            /// \param outputName If a given place is itself an operation node, this specifies name
+            /// of output port group, each group can have multiple ports
+            ///
+            /// \param outputPortIndex If place is an operational node it specifies which output
+            /// port should be considered.
+            ///
+            /// \return A vector with all operation node references that consumes data from this
+            /// place
+            virtual std::vector<Ptr> get_consuming_operations(const std::string& outputName,
+                                                              int outputPortIndex) const;
+
             /// \brief Returns a tensor place that gets data from this place; applicable for
             /// operations, output ports and output edges which have only one output port
             ///
             /// \return A tensor place which hold the resulting value for this place
             virtual Ptr get_target_tensor() const;
 
+            /// \brief Returns a tensor place that gets data from this place; applicable for
+            /// operations, output ports and output edges which have only one output port
+            ///
+            /// \param outputName Name of output port group
+            ///
+            /// \return A tensor place which hold the resulting value for this place
+            virtual Ptr get_target_tensor(const std::string& outputName) const;
```

Comment:
Do we need all these methods for our users?

Reply:
@slyalin please confirm that we need all of these methods